src/Appwrite/Task/Tasks.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Appwrite\Task;
+
+use Utopia\Platform\Service;
+
+class Tasks extends Service
+{
+    public function __construct()
+    {
+        $this->type = self::TYPE_CLI;
+        $this
+            ->addAction(Version::NAME, new Version())
```

Comment:
```suggestion
            ->addAction(Version::NAME, new Version())
```

Why don't we use a regular `getName` method to be more consistent, maybe it can be static?

Reply:
updated